src/commands/emoji.ts

```diff
@@ -0,0 +1,43 @@
+import { DiscordAPIError, EmbedBuilder, SlashCommandBuilder } from 'discord.js';
+import * as logger from '../logger';
+
+const EmojiName = 'emojiname';
+const ShouldRespondEphemeral = 'respondephemeral';
+
+const builder = new SlashCommandBuilder()
+	.setName('emoji')
+	.setDescription('Responds with the picture used for the mentioned emoji')
+	.addStringOption(option =>
+		option.setName(EmojiName).setDescription('Name of the emoji to search for').setRequired(true)
+	)
+	.addBooleanOption(option =>
+		option
+			.setName(ShouldRespondEphemeral)
+			.setDescription(
+				'When set to false this will respond with the emoji image so everyone can see. Default is true'
+			)
+	);
+
+export const emoji: GlobalCommand = {
+	info: builder,
+	requiresGuild: false,
+	async execute({ reply, interaction }): Promise<void> {
+		const emojiName = interaction.options.getString(EmojiName, true);
+		const shouldRespondEphemeral =
+			interaction.options.getBoolean(ShouldRespondEphemeral) !== null
+				? interaction.options.getBoolean(ShouldRespondEphemeral, true)
+				: true; // This slash command defaults to sending an ephemeral message
```

Comment:
This is a perfect use case for [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing):

```suggestion
		const shouldRespondEphemeral = interaction.options.getBoolean(ShouldRespondEphemeral) ?? true;
```